src/components/molecules/ImageInput/ImageInput.scss

```diff
@@ -0,0 +1,5 @@
+.image-input.default-container.image-input.default-container {
+  &--hidden {
+    display: none;
+  }
```

Comment:
There should be `mod--hidden` (`mod` as a generic term for any element) class already defined so it will not be re-defined over and over in situations like this.

Check `modifiers.scss`:

```scss
.mod {
  &--not-implemented,
  &--hidden {
    display: none;
    visibility: hidden;
  }
```

Reply:
fixed 65d922a